content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -0,0 +1,377 @@
+    $ kubectl create --context=$CTX_LOCAL -f helloworld-gateway.yaml -n sample
+    {{< /text >}}
+
+### Split Horizon EDS in action
+
+We will call the `helloworld.sample` service from another in-mesh `sleep` service.
+
+1. Deploy the `sleep` service:
+
+    {{< text bash >}}
+    $ kubectl create --context=$CTX_LOCAL -f samples/sleep/sleep.yaml -n sample
+    {{< /text >}}
+
+1. Call the `helloworld.sample` service several times:
+
+    {{< text bash >}}
+    $ kubectl exec --context=$CTX_LOCAL -it -n sample sleep-57f9d6fd6b-q4k4h -- curl helloworld.sample:5000/hello
+    {{< /text >}}
+
+If set up correctly the traffic to the `helloworld.sample` service will be distributed between the local and the remote instances resulting in responses with either `v1` or `v2` body:
+
+{{< text bash >}}
+$ kubectl exec --context=$CTX_LOCAL -it -n sample sleep-57f9d6fd6b-q4k4h -- curl helloworld.sample:5000/hello
+Defaulting container name to sleep.
+Use 'kubectl describe pod/sleep-57f9d6fd6b-q4k4h -n sample' to see all of the containers in this pod.
+Hello version: v2, instance: helloworld-v2-758dd55874-6x4t8
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl exec --context=$CTX_LOCAL -it -n sample sleep-57f9d6fd6b-q4k4h -- curl helloworld.sample:5000/hello
+Defaulting container name to sleep.
+Use 'kubectl describe pod/sleep-57f9d6fd6b-q4k4h -n sample' to see all of the containers in this pod.
+Hello version: v1, instance: helloworld-v1-86f77cd7bd-cpxhv
+{{< /text >}}
+
+You can also verify the IP addresses used to access the endpoints by printing the log of the sleep's `istio-proxy` container.
+
+{{< text bash >}}
+$ kubectl logs --context=$CTX_LOCAL -n sample sleep-57f9d6fd6b-q4k4h istio-proxy
+[2018-11-25T12:37:52.077Z] "GET /hello HTTP/1.1" 200 - 0 60 190 189 "-" "curl/7.60.0" "6e096efe-f550-4dfa-8c8c-ba164baf4679" "helloworld.sample:5000" "192.23.120.32:443" outbound|5000||helloworld.sample.svc.cluster.local - 10.20.194.146:5000 10.10.0.89:59496 -
+[2018-11-25T12:38:06.745Z] "GET /hello HTTP/1.1" 200 - 0 60 171 170 "-" "curl/7.60.0" "6f93c9cc-d32a-4878-b56a-086a740045d2" "helloworld.sample:5000" "10.10.0.90:5000" outbound|5000||helloworld.sample.svc.cluster.local - 10.20.194.146:5000 10.10.0.89:59646 -
+{{< /text >}}
+
+I.e. `192.23.120.32:443` of the remote gateway when reached v2 and `10.10.0.90:5000` of the local instance when reached v1.
+
+## Cleanup
+
+Execute the commands below to clean the demo services __and__ the Istio components.
+
+Cleanup the remote cluster:
+
+{{< text bash >}}
+$ kubectl delete --context=$CTX_REMOTE -f istio-remote-auth.yaml
+$ kubectl delete --context=$CTX_REMOTE ns istio-system
+$ kubectl delete --context=$CTX_REMOTE -f helloworld-v2.yaml -n sample
+{{< /text >}}
+
+Cleanup the local cluster:
+
+{{< text bash >}}
+$ kubectl delete --context=$CTX_LOCAL -f istio-auth.yaml
+$ kubectl delete --context=$CTX_LOCAL ns istio-system
+$ kubectl delete --context=$CTX_LOCAL -f helloworld-v1.yaml -n sample
```

Comment:
do we need to add instruction to clean up CRDs?

Reply:
It's kind of redundant if you delete the `istio-system` namespace but I don't mind adding it.

Reply:
Is it?   the CRDs are not namespace scoped, so I was not sure if removing istio-system ns would remove it.

Reply:
Yes, you are right.